sys/shell/commands/sc_at30tse75x.c

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright (C) 2015 Daniel Krebs
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_shell_commands
+ * @{
+ *
+ * @file
+ * @brief       Provides shell commands to test AT30TSE75x temperature sensor
+ *
+ * @author      Daniel Krebs <github@daniel-krebs.net>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include "at30tse75x.h"
+#define MODULE_AT30TSE75X
```

Comment:
Run make eclipsesym to create an eclipse symbol xml file that you can
import into your eclipse project. There's a page in the wiki about it.
On Oct 2, 2015 12:17 PM, "Daniel Krebs" <notifications@github.com> wrote:

> In sys/shell/commands/sc_at30tse75x.c
> <https://github.com/RIOT-OS/RIOT/pull/3755#discussion_r41007887>:
>
> > + * @{
> > + *
> > + * @file
> > + * @brief       Provides shell commands to test AT30TSE75x temperature sensor
> > + *
> > + * @author      Daniel Krebs <github@daniel-krebs.net>
> > + *
> > + * @}
> > + */
> > +
> > +#include <stdio.h>
> > +#include <string.h>
> > +#include <stdlib.h>
> > +#include <stdbool.h>
> > +#include "at30tse75x.h"
> > +#define MODULE_AT30TSE75X
>
> This slipped through by accident. Eclipse doesn't know those defines, so
> auto-completion doesn't work inside unknown #ifdefs.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/RIOT-OS/RIOT/pull/3755/files#r41007887>.
>